### ISSUE TITLE
VERSO package

### ISSUE THREAD

**Issue by danro9685** [Bioconductor/Contributions#1668]
- Repository: https://github.com/BIMIB-DISCo/VERSO

Confirm the following by editing each check box to '[x]'

- [x] I understand that by submitting my package to _Bioconductor_,
  the package source and all review commentary are visible to the
  general public.

- [x] I have read the _Bioconductor_ [Package Submission][2]
  instructions. My package is consistent with the _Bioconductor_
  [Package Guidelines][1].

- [x] I understand that a minimum requirement for package acceptance 
  is to pass R CMD check and R CMD BiocCheck with no ERROR or WARNINGS. 
  Passing these checks does not result in automatic acceptance. The 
  package will then undergo a formal review and recommendations for 
  acceptance regarding other Bioconductor standards will be addressed.

- [x] My package addresses statistical or bioinformatic issues related
  to the analysis and comprehension of high throughput genomic data.

- [x] I am committed to the long-term maintenance of my package. This
  includes monitoring the [support site][3] for issues that users may
  have, subscribing to the [bioc-devel][4] mailing list to stay aware
  of developments in the _Bioconductor_ community, responding promptly
  to requests for updates from the Core team in response to changes in
  _R_ or underlying software.

I am familiar with the essential aspects of _Bioconductor_ software
management, including:

- [x] The 'devel' branch for new packages and features.
- [x] The stable 'release' branch, made available every six
      months, for bug fixes.
- [x] _Bioconductor_ version control using [Git][5]
  (optionally [via GitHub][6]).

For help with submitting your package, please subscribe and post questions
to the [bioc-devel][4] mailing list.

[1]: https://bioconductor.org/developers/package-guidelines/
[2]: https://bioconductor.org/developers/package-submission/
[3]: https://support.bioconductor.org
[4]: https://stat.ethz.ch/mailman/listinfo/bioc-devel
[5]: http://bioconductor.org/developers/how-to/git/
[6]: http://bioconductor.org/developers/how-to/git/sync-existing-repositories/

**Comment by mtmorgan** [comment]
This issue is submitted by @danro9685 who seems to be `person("Daniele","Ramazzotti",role=c("aut"),email="daniele.ramazzotti@unimib.it")` but the maintainer (`"cre"`) is listed as `person("Davide","Maspero",role=c("cre","aut"),email="d.maspero@campus.unimib.it")`; the maintainer and the person submitting the issue should be the same. Can you please update your repository and post a comment here when corrected?

**Comment by DavideMaspero** [comment]
Dear @mtmorgan ,

I am the mantainer so I will submit the package in a new issue.

Thank you

**Comment by danro9685** [comment]
I will close this issue as @DavideMaspero will submit the VERSO package in a new issue as he is the maintainer.

**Comment by danro9685** [comment]
Dear @mtmorgan, 

we cannot open a new issue for VERSO as the system see this old one. 

Please let me and @DavideMaspero know how to proceed with this. 

Thanks and sorry for the inconvenient. 
Daniele & Davide

**Comment by mtmorgan** [comment]
@lsep can you please remove VERSO from the package database

@danro9685 once this is done, you should be able to close this issue and @DavideMaspero open a new one

Sorry for the extra effort

**Comment by DavideMaspero** [comment]
@mtmorgan 

I am tring to open a new issue, but I still receive the same error. 
Please let me know when I can submit the package, as the deadline is today.

Thank you for your collaboration,
Davide

**Comment by mtmorgan** [comment]
@DavideMaspero you'll have to wait until @lshep (sorry, tagged some random person earlier!) updates things before proceeding; @lshep will post here when ready.

**Comment by lshep** [comment]
@DavideMaspero  could you please try and open a new issue now.  I believe the issue should be resolved.

**Comment by DavideMaspero** [comment]
Dear @lshep, 

thanks a lot, I just opened the new issue as requested (see https://github.com/Bioconductor/Contributions/issues/1689). 
Please let me know if all is good now. 

Thanks, 
Davide
